.changeset/hungry-insects-push.md

```diff
@@ -6,6 +6,6 @@
 "section": tree
 ---
 
-`TreeBranchEvents` exposes `rootChanged`
+TreeBranchEvents exposes rootChanged
```

Comment:
We have a mix of present and past tense between the changesets' headings. Do we have a preference between the two?

Reply:
Inconsistencies are my fault. We want present tense since we are describing changes that are in the current release. A rule of thumb I use is to prefix the heading with "In this release," and try to ensure it reads well. The "voice" [the vscode notes](https://code.visualstudio.com/updates/v1_98) use is pretty close to my preference.